packages/firestore/test/unit/specs/existence_filter_spec.test.ts

```diff
@@ -626,9 +627,183 @@ describeSpec('Existence Filters:', [], () => {
         // Doc0 to doc49 are deleted in the next sync.
         .watchFilters([query1], docKeys.slice(0, 50), bloomFilterProto)
         .watchSnapshots(2000)
-        // BloomFilter correctly identifies docs that deleted, skip full query.
+        // Bloom Filter correctly identifies docs that deleted, skips full query.
         .expectEvents(query1, { fromCache: true })
         .expectLimboDocs(...docKeys.slice(50))
     );
   });
+
+  specTest(
+    'Resume a query with bloom filter when there is no document changes',
+    [],
+    () => {
+      const query1 = query('collection');
+      const docA = doc('collection/a', 1000, { v: 1 });
+      const bloomFilterProto = generateBloomFilterProto({
+        contains: [docA],
+        notContains: []
+      });
+      return (
+        spec()
+          .userListens(query1)
+          .watchAcksFull(query1, 1000, docA)
+          .expectEvents(query1, { added: [docA] })
+          .disableNetwork()
+          .expectEvents(query1, { fromCache: true })
+          .enableNetwork()
+          .restoreListen(query1, 'resume-token-1000')
+          .watchAcks(query1)
+          // Nothing happened while this client was disconnected.
+          // Bloom Filter includes the documents that match the query since the resume token.
```

Comment:
nit: Reword this comment:

> Bloom Filter includes the documents that match the query since the resume token

The wording "match the query since the resume token" doesn't make sense.

Maybe just say

> Bloom Filter includes the all documents that match the query, both those that have changed since the resume token and those that have not.

Whatever change you make, please make it in the other places where this comment exists in this file.